code/modules/hydroponics/hydroponics.dm

```diff
@@ -399,6 +399,11 @@
 /obj/machinery/hydroponics/proc/hardmutate()
 	mutate(4, 10, 2, 4, 50, 4, 10, 3)
 
+/obj/machinery/hydroponics/proc/glowmutate()
+	if(!myseed)
```

Comment:
`glowmutate()` is already called in a proc that checks if myseed exists. redundant.

Reply:
I do need, actually.